Shark Cage/createACLinNewDesktop/createACLinNewDesktop/util.cpp

```diff
@@ -4,121 +4,121 @@
 #include <lmerr.h>
 #include <stdexcept>
 #include <Sddl.h>
-//#include <Wdm.h>
-//#include <Ntddk.h>
-//#include <Ntifs.h>
-#include <Ntdef.h>
 #include <Wtsapi32.h>
-
-//#include <ntsecapi.h>
-
+#include <Winternl.h>
+#include <ntstatus.h>
 #include <iostream>
 #include <stdio.h>
 #pragma comment(lib, "netapi32.lib")
-//#pragma comment(lib, "Wtsapi32.lib")
 
 class tokenStructures {
 public:
-	ACCESS_MASK			AccessMask; //OK
-	POBJECT_ATTRIBUTES	ObjectAttributes; //OK
-	TOKEN_TYPE			TokenType; //OK
-	PLUID				AuthenticationId; //OK
-	PLARGE_INTEGER		ExpirationTime; //OK
-	PTOKEN_USER         TokenUser; //OK
-	PTOKEN_GROUPS       TokenGroups; //OK
-	PTOKEN_PRIVILEGES   TokenPrivileges; //OK
-	PTOKEN_OWNER        TokenOwner; //OK
-	PTOKEN_PRIMARY_GROUP TokenPrimaryGroup; //OK
-	PTOKEN_DEFAULT_DACL TokenDefaultDacl; //OK
-	PTOKEN_SOURCE       TokenSource; //OK
+	ACCESS_MASK			AccessMask;

```

Comment:
Is there a system behind the indentation 😇?

Reply:
Actually, there is 😉 It´s just GitHub UI messing it up.
![intendation system](https://user-images.githubusercontent.com/16115043/42209430-e1c418fc-7eae-11e8-9f5d-8b1ca4c0bce5.png)